tmt/utils.py

```diff
@@ -184,6 +184,13 @@ def _fmf_context(self):
         except AttributeError:
             return dict()
 
+    def steps(self):
```

Comment:
IMHO this method is not necessary (and doen't make sense for Test object). Is the only usage for fmf-id property?